src/server/main_service.cc

```diff
@@ -1056,6 +1043,67 @@ vector<bool> DetermineKeyShards(CmdArgList keys) {
   return out;
 }
 
+optional<ScriptMgr::ScriptParams> LoadScipt(string_view sha, ScriptMgr* script_mgr,
+                                            Interpreter* interpreter) {
+  auto ss = ServerState::tlocal();
+
+  if (!interpreter->Exists(sha)) {
+    auto script = script_mgr->Find(sha);
```

Comment:
not clear from reading the code what's this "script" is.  it it params?

Reply:
Both, I'll call it script_data